torch/_dynamo/variables/misc.py

```diff
@@ -123,13 +128,17 @@ def call_method(
         if is_original_tensor_torch_function:
             # Instead of tracing inside torch.Tensor.__torch_function__,
             # record the `call_function` or `call_method` call into the graph.
-            from . import TorchVariable
+            from . import ConstantVariable, ConstDictVariable, TorchVariable
             from .builder import wrap_fx_proxy
 
             original_torch_or_getattr_variable = args[0]
             new_args = args[2].items
-            new_kwargs = args[3].items
-            options = VariableTracker.propagate(self, new_args, new_kwargs.values())
+            if not isinstance(args[3], ConstantVariable):
+                new_kwargs = args[3].items
+                options = VariableTracker.propagate(self, new_args, new_kwargs)
+            else:
+                new_kwargs = ConstDictVariable(dict(), dict).items
```

Comment:
I do not understand. This seems to say "we are going to completely ignore the contents of args[3] if it is constant." But can't you have a constant that is non-empty?

Reply:
BRUH https://github.com/pytorch/pytorch/pull/109556#discussion_r1346710767